### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by zoyatanweer**
- Added homepage.
- Files to check:
   - pages/Homepage/Homepage.jsx
   - pages/Homepage/Homepage.css
   - components/Sidebar/Sidebar.jsx
   - components/Sidebar/Sidebar.css

### CODE REVIEW

File: src/pages/Homepage/Homepage.css [hunk 1]
```diff
@@ -2,36 +2,28 @@
...
   border-radius: 5px;
-  color: white;
-  color: var(--background-color);
+  color: var(--text-color);
   cursor: pointer;
   font-size: 2rem;
   height: 5rem;
   width: 100%;
   margin-bottom: 4rem;
   padding: 3px 20px;
```
Comment (Jia-A) [inline src/pages/Homepage/Homepage.css]: You can change the 20px to rem if you want.

**Review by Jia-A — APPROVED** [review]
Everything looks good.
Nice use of CSS variables throughout the code.
UI looks awesome. 
Good to merge.

File: src/pages/Homepage/Homepage.css (hunk 1 above)
Comment (Megha-Joshi) [reply src/pages/Homepage/Homepage.css]: +1

**Review by Megha-Joshi — APPROVED** [review]
- UI looks amazing.
- Looks good to me.
- Can be merged.

File: src/pages/Homepage/Homepage.css (hunk 1 above)
Comment (zoyatanweer) [reply src/pages/Homepage/Homepage.css]: done

**Pull request merged**